validation/uniqueness_test.go

```diff
@@ -0,0 +1,196 @@
+package validation
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+	"testing"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type uniqueProperties struct {
+	Name      string
+	ShortName string
+	RPC       []string
+}
+
+type chainIDSet map[uint64]bool
+
+func (s chainIDSet) AddIfUnique(id uint64) error {
+	if s[id] {
+		return fmt.Errorf("Chain with ID %d is duplicated", id)
+	}
+	s[id] = true
+	return nil
+}
+
+type chainNameSet map[string]bool
+
+func (s chainNameSet) AddIfUnique(name string) error {
+	if s[name] {
+		return fmt.Errorf("Chain with name %s is duplicated", name)
+	}
+	s[name] = true
+	return nil
+}
+
+type chainShortNameSet map[string]bool
+
+func (s chainShortNameSet) AddIfUnique(name string) error {
+	if s[name] {
+		return fmt.Errorf("Chain with short name %s is duplicated", name)
+	}
+	s[name] = true
+	return nil
+}
+
+func getGlobalChains() (map[uint]*uniqueProperties, error) {
+	chainListUrl := "https://chainid.network/chains_mini.json"
```

Comment:
Can you add a comment giving some description about what this webpage is and why its a good source of truth for superchain-registry?

Reply:
[39cc30d](https://github.com/ethereum-optimism/superchain-registry/pull/337/commits/39cc30def0f43200e210ee63af6dfd215f7523e8)